vscode-dotnet-runtime-extension/CHANGELOG.md

```diff
@@ -7,7 +7,10 @@ and this project adheres to [Semantic Versioning].
 
 ## [Unreleased]
 
-## [2.0.3] - 2024-02-05
+## [2.0.4]
+
+
+## [2.0.3] - 2024-03-21
```

Comment:
Why did the date change for 2.0.3?

Reply:
That's a good question. That's because I copy pasted the line and forgot to edit the date.